test/base.py

```diff
@@ -198,12 +214,42 @@ def run_and_verify_check_mode(self, conn_id):
 
         return found_catalogs
 
+    def is_ssl_handshake_error(self, exit_status):
+        """
+        Return True if the exit_status matches our expectations for a known SSL error.
+        Otherwise return False
+        """
+        tap_error_message = exit_status["tap_error_message"]
+        if all([exit_status['tap_exit_status'] == 1,
+                "HTTPSConnectionPool" in tap_error_message,
+                "Caused by SSLError" in tap_error_message,
+                "handshake" in tap_error_message.lower(),
+                "failure" in tap_error_message.lower()]):
+
+            return True
+        return False
+
+    # BUG_TDL-19985
+    @backoff.on_exception(backoff_wait_times,
+                          RetryableTapError,
+                          max_tries=3)
```

Comment:
```suggestion
    @backoff.on_exception(backoff.expo,
                          RetryableTapError,
                          max_tries=3,
                          factor=30)
```

Reply:
I believe this is the idiomatic way to backoff with a different factor. The backoff type should get the kwargs to the `on_exception` decorator passed to it when created by the library.

Reply:
Fixed